grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -878,6 +896,23 @@ public GrpcService build() {
         if (grpcHealthCheckService != null) {
             registryBuilder.addService(grpcHealthCheckService.bindService());
         }
+        if (autoCompression) {
+            intercept(new ServerInterceptor() {
```

Comment:
How about setting a `Compressor` to `responseFramer` if `autoCompression` is enabled?
https://github.com/line/armeria/blob/871d87297e4d051241589cb1ae95641cbc83f880/grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java#L190
It would be more efficient rather than adding an interceptor.

Reply:
It seems good.
I addressed it.